### PR TITLE
Feature/lineflow

### PR DESCRIPTION
**Pull request by sobamchan**
Integrate lineflow.

### CODE REVIEW

File: scripts/training/train.sh [hunk 1]
```diff
@@ -1,22 +1,3 @@
-P=~/mnt/DATA/small_parallel_enja
-CUDA_VISIBLE=2
+CUDA_VjSIBLE=0
```
Comment (yasufumy) [inline scripts/training/train.sh]: typo?

File: seq2seq_pytorch/data.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+from torch.utils.data import DataLoader
+
+
+START_TOKEN = '<s>'
+END_TOKEN = '</s>'
+PAD_TOKEN = '<pad>'
+UNK_TOKEN = '<unk>'
+
+
+def preprocess(x: List):
```
Comment (yasufumy) [inline seq2seq_pytorch/data.py]: This might be better;
```py
def preprocess(x: Tuple):
    ...
```

**Review by yasufumy — COMMENTED** [review]
I left some comments, please check these out.

File: scripts/training/train.sh (hunk 1 above)
Comment (sobamchan) [reply scripts/training/train.sh]: Yes, typo...

File: seq2seq_pytorch/data.py (hunk 2 above)
Comment (sobamchan) [reply seq2seq_pytorch/data.py]: Updated.
Thanks!

**Pull request merged**